index.html

```diff
@@ -41,10 +42,148 @@ <h2 class="secondinfo">I’m a software developer</h2>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
+    
+    <section class="works-template work2">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
+          <div class="tags tag1">Ruby on Trains</div>
+          <div class="tags tag2">Awesome minds</div>
+          <div class="tags tag3">JavaScript</div>
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
+    
+    <section class="works-template work3">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
+          <div class="tags tag1">Ruby on Trains</div>
+          <div class="tags tag2">Awesome minds</div>
+          <div class="tags tag3">JavaScript</div>
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
+    
+    <section class="works-template work4">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
+          <div class="tags tag1">Ruby on Trains</div>
+          <div class="tags tag2">Awesome minds</div>
+          <div class="tags tag3">JavaScript</div>
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
     
+    <section class="works-template work5">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
+          <div class="tags tag1">Ruby on Trains</div>
+          <div class="tags tag2">Awesome minds</div>
+          <div class="tags tag3">JavaScript</div>
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
     
+    <section class="works-template work6">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
+          <div class="tags tag1">Ruby on Trains</div>
+          <div class="tags tag2">Awesome minds</div>
+          <div class="tags tag3">JavaScript</div>
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
+
+  </div>
+
+<!--About me section-->
+  <section class="aboutme-container">
+    <h1 class="aboutme-h1">About me</h1>
+    <p class="aboutme-p"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.</p> 
+    <button class="aboutme-button">Get My Resume</button> 
```

Comment:
- [ ] Kindly add more width to the button to make it look just like the Figma design.
![Captura de pantalla_2021-09-18_01-20-48](https://user-images.githubusercontent.com/53111464/133878531-1ad79fdd-ba57-4681-b131-b8701bfb3100.png)

**Figma**
![Captura de pantalla_2021-09-18_01-21-04](https://user-images.githubusercontent.com/53111464/133878540-f0a2d6e7-e943-445c-a4c0-564455092230.png)